src/ucs/config/parser.c

```diff
@@ -1285,26 +1287,25 @@ static int ucs_config_parse_config_file_line(void *arg, const char *section,
     return 1;
 }
 
-ucs_status_t ucs_config_parse_config_file(const char *path, int override)
+void ucs_config_parse_config_file(const char *path, int override)
 {
-    ucs_status_t result = UCS_OK;
     int parse_result;
     FILE* file;
 
+    ucs_debug("Parsing config file: %s", path);
     file = fopen(path, "r");
     if (file == NULL) {
-        ucs_debug("Could not open config file: %s, skipping parsing", path);
-        return UCS_OK;
+        ucs_debug("Could not open config file, skipping");
```

Comment:
1. pls use small case for log messages
2. can reduce number of actual log messages: keep only
- open failed: "failed to open config file %s: %m"
- ini_parse failed: "failed to parse config file %s: %d (%s)" [stringify error from ini_parse if possible]
- success: "parsed config file %s"